_data/packages.yml

```diff
@@ -685,38 +722,38 @@
       created_at: '2023-05-26'
       stargazers_count: 14
       watchers_count: 14
-      open_issues_count: 11
+      open_issues_count: 8
       forks_count: 2
       documentation: https://sciform.readthedocs.io/en/stable/
       contrib_count: 2
-      last_commit: '2024-11-02'
+      last_commit: '2024-11-15'
   labels:
     - 6/pyOS-approved
 - package_name: astartes
   package_description: Better Data Splits for Machine Learning
   submitting_author:
-      name: Jackson Burns
+      name: Name
       github_username: JacksonBurns
   all_current_maintainers:
     - name: Kevin Spiekermann
       github_username: kspieks
-    - name: Himaghna Bhattacharjee
```

Comment:
This name was not listed on GitHub but I easily found it on the internet from other sources. I think we should be allowed to overwrite this field if real name is available elsewhere?

Reply:
Yes. so I think I mentioned this above, but the rule is that we update names in one place: contributors.yml. That update will propagate to this file. that way we don't have to worry about updating in two places!! 

We would use the same approach if this were a database. we'd have a person entry and then link them to various contribution types. :) Before we close this pr, let's make sure that the names that have been looked up are properly added to contributors.yml. they will then get updated in the packages.yml file in our next run of the bot! this should be working as i just tested it today!!